### PR TITLE
fix(Organization and Website schema enabled for specific pages)

### PR DESCRIPTION
**Pull request by Athira001**

### CODE REVIEW

File: src/structured-data/structured-data-tags.js [hunk 1]
```diff
@@ -284,10 +284,20 @@ export function StructuredDataTags({structuredData = {}}, config, pageType, resp
   const {articleType = ''} = publisherConfig['publisher-settings'] || {};
   const isStructuredDataEmpty = Object.keys(structuredData).length === 0;
   const enableBreadcrumbList = get(structuredData, ["enableBreadcrumbList"], true);
+  const WebOrgTagsArray = get$1(structuredData, ["WebOrgTagsArray"], []);
```
Comment (sharangj) [inline src/structured-data/structured-data-tags.js]: Please change name to orgTags

Comment (Athira001) [reply src/structured-data/structured-data-tags.js]: Done

File: src/structured-data/structured-data-tags.js [hunk 2]
```diff
@@ -266,6 +266,8 @@ function generateBreadcrumbListData(pageType = "", publisherConfig = {}, data =
  * @property {Website} website The website and search urls. See {@link Website} for an example
  * @property {Object} header Enable WPHeader tag. ex: `{cssSelector: ".header"}`
  * @property {Object} footer Enable WPFooter tag. ex: `{cssSelector: ".footer"}`
+ * @property {Array} structuredDataTags Should Organization and Website schema be enabled for specific pages. eg: `{structuredDataTags: ["section-page", "tag-page"]}`
```
Comment (nandap4790) [inline src/structured-data/structured-data-tags.js]: Fix the docs

Comment (Athira001) [reply src/structured-data/structured-data-tags.js]: done

**Pull request merged**